src/commands/logging/LogKey.ts

```diff
@@ -31,7 +31,16 @@ export class LogKey extends BaseCommand {
                     example: ["@Console#8939", "123313141413155", "Darkmattr"]
                 },
                 {
-                    displayName: "Dungeon",
+                    displayName: "Amount",
+                    argName: "keys",
```

Comment:
You can just rename this to `amt`. Then, you can rename the other arg to `keys`.